docs/pages/server-access/guides.mdx

```diff
@@ -12,4 +12,5 @@ layout: tocless-doc
 - [Visual Studio Code](./guides/vscode.mdx): How to remotely develop with Visual Studio Code and Teleport.
 - [JetBrains SFTP](./guides/jetbrains-sftp.mdx): How to use a JetBrains IDE to access SFTP with Teleport.
 - [Host User Creation](./guides/host-user-creation.mdx): How to configure Teleport to automatically create transient host users.
+- [Linux Auditing System](./guides/auditd.mdx): How to integrate Teleport with Linux Auditing System (auditd).
```

Comment:
```suggestion
- [Linux Auditing System](./guides/auditd.mdx): How to integrate Teleport with the Linux Auditing System (auditd).
```
For consistency with other docs, e.g., https://man7.org/linux/man-pages/man8/auditd.8.html